src/scripts/CommandPattern/MemoryCalculateCommand/MemoryCalculateCommand.js

```diff
@@ -0,0 +1,26 @@
+export class MemoryCalculate {
+  constructor(firstOperand, operator) {
+    this.firstOperand = firstOperand;
+    this.operator = operator;
+  }
+
+  execute() {
+    if (this.operator === '+') {
+      localStorage.setItem(
+        'number',
+        +localStorage.getItem('number') + +this.firstOperand,
```

Comment:
Смотри как выглядит нечитабельно строчка из-за того что ты приводишь к числу через +localStorage.getItem('number') ...

Используй лучше parseInt

```suggestion
        parseInt(localStorage.getItem('number')) + parseInt(this.firstOperand),
```